lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -513,6 +520,13 @@ nvme_library_init(void)
 
 	probe_ctx.controllers_remaining = num_controllers;
 
+	val = spdk_conf_section_get_val(sp, "ResetControllerOnTimeout");
```

Comment:
This new option should be documented in the example configuration files (etc/spdk/iscsi.conf.in and etc/spdk/nvmf.conf.in).